app/Http/Controllers/AdminController.php

```diff
@@ -379,50 +380,7 @@ public function upgrade()
 
         /* Cleanup the database */
         if ($Cleanup) {
-            self::delete_unused_rows('banner', 'projectid', 'project');
```

Comment:
Is the "cleanup database" button needed anymore?  I feel that we should move these kinds of administrative tasks to CLI options only.